editor.planx.uk/src/ui/public/FeedbackDisclaimer.tsx

```diff
@@ -1,23 +1,18 @@
-import Box from "@mui/material/Box";
 import Link from "@mui/material/Link";
+import Box from "@mui/material/Box";
 import Typography from "@mui/material/Typography";
 import React from "react";
 
-export default function FeedbackDisclaimer(): FCReturn {
+interface FeedbackDisclaimerProps {
+  children: React.ReactNode;
+}
+
+export default function FeedbackDisclaimer({
+  children,
+}: FeedbackDisclaimerProps): FCReturn {
```

Comment:
FYI: React has a helper for just this situation! It also accepts a generic, so it can be used to wrap existing interfaces (a search in the codebase for `PropsWithChildren` should illustrate some nice examples of this 👌)

```suggestion
import React, { PropsWithChildren } from "react";

export default function FeedbackDisclaimer({
  children,
}: PropsWithChildren): FCReturn {
```